src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
CredSSP cmdlet tests.Disable-WSManCredSSP works: Client
```powershell
The process cannot access the file because it is being used by another process. (0x80070020)
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\CredSSP.Tests.ps1: line 70
70:         Disable-WSManCredSSP -Role $role | Should -BeNullOrEmpty

```